layouts/index.html

```diff
@@ -14,56 +14,38 @@ <h1 class="mt-0">{{ .Title }}</h1>
       <h5 class="mt-0">Chat with us:</h5>
         <a href="https://matrix.to/#/#invidious:matrix.org"><img alt="Matrix" src="https://img.shields.io/matrix/invidious:matrix.org?label=Matrix&color=darkgreen"></a>
         <a href="https://web.libera.chat/?channel=#invidious"><img alt="Libera.chat (IRC)" src="https://img.shields.io/badge/IRC%20%28Libera.chat%29-%23invidious-darkgreen"></a>
+
+        <br/>
+        <br/>
+
     </div>
   </div>
 </section>
 {{ end }}
 
 {{ define "sidebar-prefooter" }}
+<hr>
 <section class="section section-sm">
   <div class="container">
     <div class="row justify-content-center text-center">
       <div class="col-lg-5">
-        <h2 class="h4">Data import/export</h2>
-        <p>
-          <ul>
-            <li>Import subscriptions from YouTube, NewPipe and Freetube
-            <li>Import watch history from NewPipe
-           <li>Export subscriptions to NewPipe and Freetube
-           <li>Import/Export Invidious user data
-          </ul>
-        </p>
+        <img src="undraw_privacy_protection_nlwy.svg" height=250/>
+        <h2 class="h4">Privacy</h2>
+        <p> With protections against the prying eyes of google, and lack of build-in tracking, Invidious help keeps your data private.  </p>
       </div>
       <div class="col-lg-5">
-        <h2 class="h4">User features</h2>
-        <p>
-          <ul>
-            <li>Lightweight
-            <li>No ads
-            <li>No tracking
-            <li>No JavaScript required
-            <li>Light/Dark themes
-            <li>Customizable homepage
-            <li>Subscriptions independant from Google
-            <li>Notifications for all subscribed channels
-            <li>Audio-only mode (with background play on mobile)
-            <li>Support for Reddit comments
-            <li>Available in many languages, thanks to our translators
-          </ul>
-        </p>
+        <img src="undraw_regain_focus_ecvj.svg" height=250/>
+        <h2 class="h4">Humanely designed</h2>
+        <p> Invidious helps you regain your focus through ethical design! No more shall you conclusively check YouTube and waste your day away!</p>
```

Comment:
We should mention that users won't be affected by "bad" features like recommendations and other "addictive" features.